src/vue/backend/src/store/modules/general/actions.js

```diff
@@ -13,7 +13,63 @@ const verifyAPIkey = () => {
     return new Promise(resolve => setTimeout(resolve, 5000))
 }
 
+const saveOmnibarSettings = () => {
+    // To-Do: add actual endpoints
+    // GET /idxbroker/v1/admin/settings/omnibar
```

Comment:
You don't need to add the api skeleton in here, just the to-do is fine. This information will not be used explicitly.